darkmode.scss

```diff
@@ -548,6 +548,11 @@
     }
   }
 
+  .tweet-form.condensed * .btn {
```

Comment:
@DylanPlemmons The `*` selector is pretty expensive on the browser. We're applying a whole stylesheet on top of Twitter's stylesheet, so there's already a slight performance hit. Is there any other way you could make this code work without using the `*` selector?